### DISCUSSION TITLE
ANSIfy the drinking squirrel gif

### DISCUSSION THREAD

**Discussion by snobu** [spectreconsole/spectre.console#726] [Ideas]
Can someone ANSIfy this drinking squirrel progress gif and make it part of the Spinners? I know it's a tough ask but it could become the cornerstone of this century's spinners.

![](https://raw.githubusercontent.com/snobu/powerbi-embed-v2/master/static/img/squirrel_loading.gif)

**Comment by phil-scott-78** [comment]
Take a look at https://khalidabuhakmeh.com/gifs-in-console-output-imagesharp-and-spectreconsole
